tests/measure-perf.bats

```diff
@@ -162,6 +162,27 @@ function get_perf_opts() {
   [ "$npng" -eq 0 ] # no detailed charts because of --skip-perf
 }
 
+@test "measure perf --skip-perf --use-vdenc --skip-ffmpeg raw h264" {
```

Comment:
why we don't test vdenc via ffmpeg? That's a gap, we must test it as we do for quality. See ffmpeg flag here:
https://github.com/intel/media-delivery/blob/c31ea6ac907b64d359b59429af8ced0105d73b19/measure/quality/measure-quality#L423

Reply:
Yes we can, its already supported for FFMPEG. Just need to modify the test, to remove the --skip-ffmpeg knob